docs/_includes/example.njk

```diff
@@ -25,8 +25,14 @@
   </ul>
 
   <div class="app-tabs__panel" id="html-default-{{ id }}" role="tabpanel" aria-labelledby="tab_html-default-{{ id }}">
-    {%- if htmlWarning -%}
-      <div class="app-example__warning">{{ htmlWarning | renderMarkdown | safe }}</div>
+    {%- if tabWarning -%}
+      <div class="govuk-warning-text app-example__warning">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-visually-hidden">Warning</span>
+          {{ tabWarning | renderMarkdown | safe }}
+        </strong>
+      </div>
```

Comment:
I thought the decision was that we weren't doing this today? But that we were going to wait and see if it was needed? Or have there been further conversations I'm not aware of?